src/webapp/src/auth/Auth.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+import { AuthStatuses } from './types';
+import { RootState } from '../reducers';
+import { signInUser, signOutUser } from './asyncAuthActions';
+
+const Loader = styled.div`
```

Comment:
I used that loader as a placeholder, I think we could make something better :D. Would you prefer if I don't make PRs with those placeholders? I didn't want to try to make a real loader because we haven't decide important styling points.